gapis/api/vulkan/resources.go

```diff
@@ -253,9 +253,9 @@ func getImageFormatFromVulkanFormat(vkfmt VkFormat) (*image.Format, error) {
 	case VkFormat_VK_FORMAT_R32_SFLOAT:
 		return image.NewUncompressed("VK_FORMAT_R32_SINT", fmts.R_F32), nil
 	case VkFormat_VK_FORMAT_B10G11R11_UFLOAT_PACK32:
-		return image.NewUncompressed("VK_FORMAT_B10G11R11_UFLOAT_PACK32", fmts.BGR_F10F11F11), nil
+		return image.NewUncompressed("VK_FORMAT_B10G11R11_UFLOAT_PACK32", fmts.RGB_F11F11F10), nil
 	case VkFormat_VK_FORMAT_E5B9G9R9_UFLOAT_PACK32:
-		return image.NewUncompressed("VK_FORMAT_E5B9G9R9_UFLOAT_PACK32", fmts.RGBE_U9U9U9U5), nil
+		return image.NewUncompressed("VK_FORMAT_E5B9G9R9_UFLOAT_PACK32", fmts.EBGR_U5U9U9U9), nil
```

Comment:
Actually, this one was correct, so this change should be undone.

I think it is worth adding a comment to all the "VK_*BGR*" -> "fmts.*RGB*" mappings to communicate that this re-ordering is due to the little endianess or our stream package, to make sure readers don't get confused and think it's a bug.

Reply:
Thanks for catching this the fix is in #1203